events/wc_events/wc_lordaeron_events.txt

```diff
@@ -0,0 +1,410 @@
+﻿namespace = lordaeron 
+
+##############################################
+# 1001 - 1099: Reclamation of Lordaeron Events
+##############################################
+
+lordaeron.1001 = { 
+
+	type = character_event
+	title = lordaeron.1001.title
+	desc = lordaeron.1001.desc
+	theme = stewardship
+	override_background = market
+
+	immediate = { 
+		title:b_lordaeron.title_province = { 
+			remove_building = ruined_palace_01
+			add_special_building = ruined_palace_02 
+		}
+		title:c_lordaeron ={
+			set_county_culture = culture:forsaken
+		}
+	}
+
+	right_portrait = { 
+		character = root
+		animation = steward
+	}
+
+	trigger = { 
+		OR = { 
+			is_of_forsaken_like_culture = yes 					#make sure i have the title...
+			title:b_lordaeron.holder = { 						#or that I gave out the title to someome...
+				self_or_liege_is_of_forsaken_like_culture = yes 
+				any_liege_or_above = {this = root }
+			}
+		}
+	}
+
+	on_trigger_fail = { 
+		trigger_event = lordaeron.1002
+	}
+
+	option = { 
+		name = lordaeron.1001.opt.a
+
+		show_as_tooltip = { 
+			custom_tooltip = lordaeron.1001.tooltip.a
+		}
+
+		hidden_effect = { 
+			global_var:sylvanas_character = { 
+				trigger_event = { # replace with on-action for redirect?? or just use trigger_fail???
+					id = lordaeron.1101 #selenite texts found in undercity; revival imminent
+					years = 2
+					delayed = yes
+				}
+			}
+			while = { 
+				count = 3 # in three different courts...
+				random_vassal = { 
+					limit = { 
+						faith.religion = { is_in_family = rf_shadow }
+						NOT = { government_has_flag = government_is_theocracy }
+					}
+					while = {
+						count = 4 # four characters will adopt Forgotten Shadow as secret faith
+						random_pool_character = { 
+							province = this.capital_province
+							limit = {
+								NAND = { 
+									has_council_position = councillor_court_chaplain
+									NOT = { faith = faith:forgotten_shadow }
+									NOT = { government_has_flag = government_is_theocracy }
+									NOT = { piety_level >= 2 }
+									NOT = { exists = secret_faith }
+									NAND = { has_trait = zealous has_trait = loyal has_trait = holy_warrior has_trait = cynical }
+								}
```

Comment:
which character flag things? `is_of_forsaken_like_culture`?

Reply:
look at the code above, `while` for vassals, i added character flag there in suggestion so it doesnt pick the same character several times. It can choose same pool character here too several times

Reply:
ah, gotcha. i think Paradox uses lists for this as i do below, i think i am gonna go ahead with that method if that works for you. i know it can't pick the same vassal more than once

Reply:
i figured out how to use a while loop so that i don't have to hard code 12 individuals or set flags. this should work and if it doesn't it will at the very worst result in less than 12 converts